src/chocolatey/infrastructure.app/services/CygwinService.cs

```diff
@@ -42,9 +42,9 @@ public sealed class CygwinService : ISourceRunner
         private readonly INugetService _nugetService;
         private readonly IFileSystem _fileSystem;
         private readonly IRegistryService _registryService;
-        private const string PACKAGE_NAME_TOKEN = "{{packagename}}";
-        private const string INSTALL_ROOT_TOKEN = "{{installroot}}";
-        public const string CYGWIN_PACKAGE = "cygwin";
+        private const string PackageNameToken = "{{packagename}}";
+        private const string InstallRootToken = "{{installroot}}";
+        private const string CygwinPackage = "cygwin";

```

Comment:
I understand the change here, for consistency, but is there an obsoletion process we should follow here, rather than a straight public to private?

Reply:
I added an obsoletion to the end of this file similar to the method obsoletions. Is there more you'd want this to do?